zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/Zab1_0Test.java

```diff
@@ -554,36 +554,41 @@ public void converseWithLeader(InputArchive ia, OutputArchive oa,
             }
         }, 2);
     }
-    
+
     // We want to track the change with a callback rather than depending on timing
     class TrackerWatcher implements Watcher {
         boolean changed;
+
         synchronized void waitForChange() throws InterruptedException {
-            while(!changed) {
+            while (!changed) {
                 wait();
             }
         }
+
         @Override
         public void process(WatchedEvent event) {
             if (event.getType() == EventType.NodeDataChanged) {
-                synchronized(this) {
+                synchronized (this) {
                     changed = true;
                     notifyAll();
                 }
             }
         }
+
         synchronized public boolean changed() {
             return changed;
         }
 
-    };
-    
+    }
+
+    ;
```

Comment:
Is this really enforced by checkstyle?

Reply:
@anmolnar  this PR will be redone for not obeying the way we discuss in the email thread